src/pg/pg-store.ts

```diff
@@ -478,6 +482,31 @@ export class PgStore extends BasePgStore {
     }
   }
 
+  async getInscriptionCountPerBlock(
+    filters?: DbInscriptionCountPerBlockFilters
```

Comment:
Would you ever need to call this function without filters? Can we just make them mandatory?